examples/quantizing_moe_fp8/README.md

```diff
@@ -0,0 +1,99 @@
+# Quantizing TinyMixtral-4x248M-MoE Model with FP8
+
+This directory contains an example script for quantizing the `TinyMixtral-4x248M-MoE` model using the FP8 quantization scheme.
+
+## Installation
+
+To get started, install the necessary dependencies by executing the following commands:
+
+```bash
+git clone https://github.com/vllm-project/llm-compressor.git
+cd llm-compressor
+pip install -e .
+```
+
+## Quickstart
+
+The provided example script demonstrates an end-to-end process for applying the quantization algorithm:
+
+```bash
+python3 mixtral_moe.py
+```
+
+## Creating a Quantized MoE Model
+
+This example leverages `LLMCompressor` and `Compressed-Tensors` to create an FP8-quantized `Mixtral-8x7B-Instruct-v0.1` model. The model is calibrated and trained using the `open_platypus` dataset.
```

Comment:
should probably just list the package names? `llm-compressor` and `compressed-tensors`

Reply:
Done!